keyboards/0xcb/splaytoraid/info.json

```diff
@@ -0,0 +1,140 @@
+{
+    "manufacturer": "Freya",
+    "keyboard_name": "splaytoraid",
+    "maintainer": "freya-irl",
+    "url": "https://github.com/freya-irl/splaytoraid40",
+    "debounce": 5,
+    "diode_direction": "COL2ROW",
+    "matrix_size": {
+        "cols": 7,
+        "rows": 8
+    },
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0xCB00",
+        "vid": "0x2004"
+    },
+    "features": {
+        "encoder_map": true,
```

Comment:
This feature should not be enabled at the keyboard level, and needs to be moved to `keymaps/*/rules.mk`.
```suggestion
```